recipes/xgboost/meta.yaml

```diff
@@ -94,6 +104,9 @@ outputs:
       string: cuda_{{ cuda_major }}_py{{ py_version }}_{{ build_number }}
       ignore_run_exports:
         - nvcc
```

Comment:
Curious why this was needed and if it is still needed in light of this change

Reply:
Seems like we can probably remove this in a future PR on branch-22.06

Reply:
Yeah neither of these is pressing. Just a couple things that stood out to me